tracer/src/Datadog.Trace/Logging/Internal/DatadogLoggingFactory.cs

```diff
@@ -166,13 +166,10 @@ private static string GetDefaultLogDirectory(IConfigurationSource? source, strin
 #else
             var isWindows = FrameworkDescription.Instance.IsWindows();
 
-            ImmutableAzureAppServiceSettings? aasSettings = null;
-            if (source != null)
-            {
-                aasSettings = new ImmutableAzureAppServiceSettings(source, NullConfigurationTelemetry.Instance);
-            }
-
-            if (aasSettings?.IsFunctionsAppConsumptionPlan == true)
+            if (
```

Comment:
nit: I think you can make `source` and `telemetry` non-null because they're non-null in the `GetLogDirectory()` function. Then you can remove the `is not null` checks here

Reply:
done!